bubblewrap.c

```diff
@@ -1491,6 +1510,14 @@ main (int    argc,
         }
 
       /* Initial launched process, wait for exec:ed command to exit */
+      if (opt_info_fd != -1)
+        {
+          char output[64];
+          int len = sprintf (output, "{\n    \"child-pid\": %i\n}\n", pid);
```

Comment:
With a fixed size buffer, at least use snprintf, this is running setuid after all.
Or use xasprintf() for an allocating, oom-handling version.